docset/winserver2012-ps/failoverclusters/Test-Cluster.md

```diff
@@ -310,17 +252,21 @@ Accept wildcard characters: False
 ```
 
 ### -Pool
-Specifies the clustered storage pool or pools for which to run the cmdlet.
-When the specified storage pool is online and a virtual disk in the storage pool is assigned to a clustered role or Cluster Shared Volume, you must also specify the **Force** parameter to take the storage pool offline for the duration of the storage tests.
-Otherwise, the specified storage pool must be taken offline before the storage tests.
-If the **Pool** parameter is not specified, storage tests run on all storage pools that are available for use in the cluster or that are in the cluster resource offline or failed state.
-The acceptable values for this parameter are:
 
- -- System.String: A string that represents the name of the clustered storage pool or pools. 
+Specifies the clustered storage pool or pools for which to run the cmdlet. When the specified
+storage pool is online and a virtual disk in the storage pool is assigned to a clustered role or
+Cluster Shared Volume, you must also specify the **Force** parameter to take the storage pool
+offline for the duration of the storage tests. Otherwise, the specified storage pool must be taken
+offline before the storage tests. If the **Pool** parameter is not specified, storage tests run on
```

Comment:
Clarification question: If I don't specify **Force** and the disks are online, does the cmdlet error?

Reply:
Good questions, I'm just waiting on the dev team to confirm

Reply:
Wording updating to reflect the tested behaviour.